python/invoke_release/tasks.py

```diff
@@ -1045,6 +1049,67 @@ def _post_rollback(current_version, rollback_to_version):
         plugin.post_rollback(ROOT_DIRECTORY, current_version, rollback_to_version)
 
 
+def _get_version_to_bump(changelog_message):
+    major_commit_present = None
+    minor_commit_present = None
+    patch_commit_present = None
+
+    for line in changelog_message:
+        if line.startswith('- ' + MAJOR_TAG):
+            major_commit_present = True
```

Comment:
If a major commit is present, you can just return "major" immediately without continuing the loop, because at that point it does not matter what any of the other commits are, even if they _don't_ have prefixes: It will have to be a major bump.

Reply:
Done. Thank you.